src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+
+  --color: #00ACDC;
 }
+
+
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
```

Comment:
add `align-items: center` to make give logo appropriate spaces on top and bottom instead of using margin on the logo element